app/javascript/src/Stats/lib/metric.jsx

```diff
@@ -4,10 +4,15 @@ export class StatsMetric {
   }
 
   _buildMetric (metric) {
+    let systemName = metric.system_name
+    let parentId = metric.parent_id
+
     this.id = metric.id
     this.name = metric.friendly_name
     this.serviceId = metric.service_id
-    this.systemName = metric.system_name
-    this.isMethod = metric.parent_id != null
+    this.systemName = systemName
+    this.parentId = parentId
+    this.isMethod = parentId != null
+    this.isHits = !!systemName.match(/hits(\.\d+)?/)
```

Comment:
I've seen this RegEx used in many other files, maybe we could inside this file also export a `const` with it... like `HITS_METRIC` or something